kubeflow/trainer/rhai/transformers.py

```
@@ -323,9 +324,88 @@ def checkpoint_in_progress(self):
+                    self.fsspec.info(f"{self.cloud_storage_protocol}://{bucket}")
+                    print(
+                        f"[Kubeflow] Cloud storage configured: {cloud_remote_storage_uri} "
+                        f"({self.cloud_storage_protocol})",
+                        flush=True,
+                    )
+                except FileNotFoundError as e:
+                    raise RuntimeError(
+                        f"Bucket '{bucket}' not found in {self.cloud_storage_protocol} storage. "
+                        f"Please verify the bucket exists and the URI is "
+                        f"correct: {cloud_remote_storage_uri}"
+                    ) from e
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)